tests/integration/parsers/apps/aiida/test_parser.py

```diff
@@ -0,0 +1,18 @@
+from tests.integration import IntegrationTestBase
+from express.parsers.apps.aiida.parser import AiidaArchiveParser
+
+
+class TestAiidaArchiveParser(IntegrationTestBase):
+
+    def setUp(self):
+        super(TestAiidaArchiveParser, self).setUp()
+        self.parser = AiidaArchiveParser(workDir=self.workDir)
+
+    def test_aiida_num_structures(self):
```

Comment:
It's somewhat cryptic what we are testing here. Let's either have a more explanatory name or explain in the docstring, pls.